src/components/TodoContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import TodosList from "./TodosList";
+
+class TodoContainer extends React.Component {
+
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Setup development environment",
+        completed: true
+      },
+      {
+        id: 2,
+        title: "Develop website and add content",
+        completed: false
+      },
+      {
+        id: 3,
+        title: "Deploy to live server",
+        completed: false
+      }
+    ]
+   };
+
+   render() {
+    return (
+      <div>
+        <TodosList todos={this.state.todos} />
```

Comment:
Hello, I notice your application is starting with compiled errors, It is important to fix these errors for smooth app execution, Kindly fix the errors, and ensure all linter checks pass. Refer to the react linter documentation here [React-linter-configuration](https://github.com/microverseinc/linters-config/tree/master/react-redux) for guidance on resolving compiled errors.

![start-errors](https://github.com/Victoria129/react-to-do-app/assets/100206273/22b32fb6-e0ea-4ea6-a39a-023b2004dfb8)